### ISSUE TITLE
[Question] Expand tax summary on invoice - polish requirements

### ISSUE THREAD

**Issue by csych** [YetiForceCompany/YetiForceCRM#13116]
I'm trying to adjust invoice template according to polish law requirements. That's why I need a bit more complex tax summary. I took a look at TableTaxSummary.php but unfortunately I won't be able to code it by myself.

At the moment tax summary on invoice looks like this (example with one tax rate only):
23% | 138 zł
------------ | -------------
**Amount** | **138 zł**

I need it to look like this:
Tax rate | Net | Tax | Gross
------------ | ------------- | ------------- | -------------
23% | 600 zł | 138 zł | 738 zł
**Amount** | **600 zł** | **138 zł** | **738 zł**

So basically it should contain Net and Gross as well.

Does anyone have similar solution already implemented and could share?
I would appreciate any help.

**Comment by fbojan** [comment]
I myself have such a problem, because Slovenia needs to show both amounts.

**Comment by jmginfor** [comment]
Same here. And multiple taxes too.

**Comment by bpabiszczak** [comment]
This is our standard invoice generated in YetiForce, what is missing in it?

![image](https://user-images.githubusercontent.com/10197525/91489351-fde8c500-e8b0-11ea-9411-6f6df8f9aad0.png)

**Comment by vdpawel** [comment]
You can easily achieve this goal by adding appropriate functions in the app / TextParser or need a programmer that you will do it
![obraz](https://user-images.githubusercontent.com/62466142/91491153-ef4fdd00-e8b3-11ea-99f7-4ae830d84429.png)

**Comment by fbojan** [comment]
> You can easily achieve this goal by adding appropriate functions in the app / TextParser or need a programmer that you will do it
> ![obraz](https://user-images.githubusercontent.com/62466142/91491153-ef4fdd00-e8b3-11ea-99f7-4ae830d84429.png)
Where could I get this TextParser / TableTaxSummary.php and ProductsTableLongVersion.php because that's exactly what I need.  If I have to pay let me know.

Thanks

**Comment by vdpawel** [comment]
in the app / text parser directory you have ready-made examples to use :)
My code is not good enough to share and I count the tax from the net amount, but if you can't program it by yourself, you can buy the support
here you are
#13279
